meta.yaml

```diff
@@ -30,6 +30,7 @@ requirements:
     - jsonschema
     - noaa-gfdl::catalogbuilder
     - noaa-gfdl::fre-nctools
+    - noaa-gfdl::analysis_scripts
```

Comment:
this needs to be in `environment.yml` as well, for tests/building in `.github/workflows/create_test_conda_env.yaml`

Reply:
@ilaflott Thank you, I have added that to the `environment.yml`.

Reply:
Oh I see, `fre/tests` (not `fre/test`), my bad

Reply:
I'm not sure how to add it to the `setup.py`, since it is in a subdirectory of a github repo.  I'll try to do some research.

Reply:
ah, we dont have `catalogbuilder` there either. must be something about the way the package's metadata gets set, but I don't know exactly what. Not important- doesn't affect functionality. `conda install` picks up the slack just fine, presumably because the equivalent info is in `meta.yaml`